src/main/java/org/littil/api/ApplicationLifeCycle.java

```diff
@@ -1,33 +1,39 @@
 package org.littil.api;
 
 import com.auth0.client.mgmt.ManagementAPI;
+import com.auth0.client.mgmt.filter.UserFilter;
 import com.auth0.exception.Auth0Exception;
 import com.auth0.json.mgmt.users.User;
 import io.quarkus.runtime.StartupEvent;
 import io.quarkus.runtime.configuration.ProfileManager;
 import lombok.extern.slf4j.Slf4j;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
-import org.littil.api.auth.provider.Provider;
 import org.littil.api.auth.service.AuthorizationType;
+import org.littil.api.guestTeacher.service.GuestTeacherService;
+import org.littil.api.school.service.SchoolService;
+import org.littil.api.user.service.UserService;
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.event.Observes;
 import javax.inject.Inject;
-import javax.persistence.PersistenceException;
-import javax.sql.DataSource;
 import javax.transaction.Transactional;
-import java.sql.Connection;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
-import java.util.Map;
-
-import static java.util.Collections.emptyList;
+import java.util.*;
+import java.util.stream.Stream;
 
 @ApplicationScoped
 @Slf4j
 public class ApplicationLifeCycle {
 
+    //TODO: make this configurable for other purposes (e2e tests for example)
```

Comment:
vervolgstap is om hiervoor een config thingie te maken zodat het configureerbaar is voor b.v. e2e

Reply:
Eens. Pakken we later op.